lua/astrocommunity/git/mini-git/init.lua

```diff
@@ -0,0 +1,8 @@
+---@type LazySpec
+return {
+  "echasnovski/mini-git",
+  event = "User AstroFile",
```

Comment:
```suggestion
  event = "User AstroGitFile",
```

only load automatically for git files